build/localnet/Tiltfile

```diff
@@ -105,19 +115,19 @@ COPY bin/client-linux /usr/local/bin/client
 # Pushes localnet manifests to the cluster.
 k8s_yaml(
     [
-        "build/localnet/private-keys.yaml",
-        "build/localnet/configs.yaml",
-        "build/localnet/cli-client.yaml",
-        "build/localnet/network.yaml",
+        "manifests/private-keys.yaml",
+        "manifests/configs.yaml",
+        "manifests/cli-client.yaml",
+        "manifests/network.yaml",
         local(
-            "build/localnet/v1-validator-template.sh %s"
+            "templates/v1-validator-template.sh %s"
             % localnet_config["validators"]["count"],
             quiet=True,
         ),
     ]
 )
 
-# # Exposes postgres port to 5432 on the host machine.
+# An example of how to expose postgres port to 5432 on the host machine in case developer wants to connect to postgres directly
```

Comment:
```suggestion
# Uncomment in order to expose postgres port to 5432 on the host machine in case the developer wants to connect to postgres directly
```

Reply:
Question: Would this conflict with the postgres that's managed by Tilt?

Reply:
This is to expose the postgres database managed by tilt. But it might conflict with any other postgres that runs on your machine. I mentioned how to avoid that in my recent edit.